docs/development/accessibility.md

```diff
@@ -0,0 +1,120 @@
+# Accessibility
+
+In the context of software development accessibility refers to
+the ability of people with disabilities to effectively
```

Comment:
```suggestion
the use of assistive technologies to aid users in effectively
```

accessibility features are for everyone, not only for "disabled people" - see [microsoft's inclusivity kit](https://download.microsoft.com/download/b/0/d/b0d4bf87-09ce-4417-8f28-d60703d672ed/inclusive_toolkit_manual_final.pdf)
<img width="608" alt="Screen Shot 2022-05-04 at 16 40 29" src="https://user-images.githubusercontent.com/1466420/166693409-97ea3edb-6d73-4e5a-8241-e63ee10365dc.png">